testscommon/components/components.go

```diff
@@ -173,7 +173,7 @@ func GetConsensusArgs(shardCoordinator sharding.Coordinator) consensusComp.Conse
 		StatusComponents:     statusComponents,
 		StatusCoreComponents: GetStatusCoreComponents(),
 		ScheduledProcessor:   scheduledProcessor,
-		SubroundBlockType:    consensus.SubroundBlockTypeV1,
+		ConsensusModel:       consensus.ConsensusModelV1,
```

Comment:
For the next PR: can we have the consensus integration tests running on both variants? It will be way easier to debug if we need to in the future